prov/gni/test/api.c

```diff
@@ -297,16 +300,18 @@ int rdm_api_check_data(char *buf1, char *buf2, int len)
 
 void rdm_api_check_cqe(struct fi_cq_tagged_entry *cqe, void *ctx,
 		      uint64_t flags, void *addr, size_t len,
-		      uint64_t data)
+		      uint64_t data, struct fid_ep *fid_ep)
```

Comment:
Why are we maintaining this function if it's not used?  In fact, all the rdm_api_check_* functions are not used and should be removed.

@chuckfossen Is there a reason to keep them around?